mmcls/apis/inference.py

```diff
@@ -64,6 +64,8 @@ def inference_model(model, img):
     test_pipeline = Compose(cfg.data.test.pipeline)
     # prepare data
     data = dict(img_info=dict(filename=img), img_prefix=None)
+    test_load = LoadImageFromFile()
```

Comment:
We should support the input parameter ``img`` to be either ``np.ndarry`` or ``str``.
So to improve the code:
1. in docstring: change the description of ``img``
2. use ``isinstance(...)`` to check the type of ``img``. If it is ``str``, load image and execute pipeline; else, use another pipeline.